cmd/containerboot/serve.go

```diff
@@ -127,6 +133,10 @@ func readServeConfig(path, certDomain string) (*ipn.ServeConfig, error) {
 	if err != nil {
 		return nil, err
 	}
+	if len(j) == 0 {
```

Comment:
When do we expect this condition to happen? May be worth a code comment as it seems a bit surprising on first glance.

Reply:
Not with the operator config, but users could have well loaded empty config (i.e to be updated later).
I _think_ I hit that as I was experimenting with non-operator serve config

Reply:
(added a comment though)